rbi/core/regexp.rbi

```diff
@@ -1129,6 +1129,20 @@ class Regexp < Object
   end
   def match(arg0, arg1=T.unsafe(nil), &blk); end
 
+  # Returns a true or false indicates whether the regexp is matched or
+  # not without updating $~ and other related variables. If the second
+  # parameter is present, it specifies the position in the string to
+  # begin the search.
+  # https://ruby-doc.org/core-2.7.0/Regexp.html#method-i-match-3F
+  sig do
+    params(
+        arg0: String,
```

Comment:
I found 4 places in our codebase that were passing a `Symbol` to arg0, and 15 places that were passing a nilable string to `match?`. Neither `//.match?(nil)` nor `//.match?(:"")` raises an exception in Ruby. What do you think about relaxing this type to `T.nilable(T.any(String, Symbol))`?

Reply:
Sounds good. Passing a symbol looks pretty weird to me but I guess it's valid.